gradle/check-licenses.gradle

```diff
@@ -18,6 +18,7 @@ ext.acceptedLicenses = [
     'BSD License',
     'BSD 2-Clause',
     'BSD 3-Clause',
+    'The JSON License',
```

Comment:
Just to be safe I would double-check if we indeed accept that license

Reply:
Good point, asked in prod dev as a starting point.

Reply:
I've had a chat and made sure this is ok, just need to also make sure we pull in the license etc. I'll follow up more today.